virtual-users/src/main/kotlin/com/atlassian/performance/tools/hardware/vu/BrowseBoardsAction.kt

```diff
@@ -27,7 +27,7 @@ class PatientBrowseBoardsAction(
 
     override fun run() {
         val browseBoardsPage = meter.measure(BROWSE_BOARDS) {
-            jiraSoftware.goToBrowseBoards().waitForBoardsList(Duration.ofMinutes(2))
+            jiraSoftware.goToBrowseBoards().waitForBoardsList(Duration.ofMinutes(10))
```

Comment:
In the 10 min result:
https://s3-eu-west-1.amazonaws.com/quicksilver-jhwr-cache-ireland/QUICK-132-fix-v3/c5.18xlarge/nodes/5/dbs/m4.4xlarge/sub-test-comparison/summary-per-cohort.html
the average response time is 52645 and 53762. which is way below than 10 mins. I checked the actual result and did not find one success action exceed 1 min.

In Summary:
1. the 10 min timeout has better result, much lower error rate
2. however, it seems 10 min timeout is not necessary, because the actual time spend is within 1 min (@dagguh has suspected maybe there is a client-side timeout as 1 min).

We can park this issue and merge/continue our HWR or maybe it's worth to do another experiment?

Reply:
okay, let's merge it and I will raise another ticket to track this one.

Reply:
### 10 minutes timeout
[Distro](https://s3-eu-west-1.amazonaws.com/quicksilver-jhwr-cache-ireland/QUICK-132-fix-v3/c5.18xlarge/nodes/5/dbs/m4.4xlarge/sub-test-comparison/distribution-comparison.html): 30% hit the 10 minute timeout, the rest is at 1 minute
[Timeline](https://s3-eu-west-1.amazonaws.com/quicksilver-jhwr-cache-ireland/QUICK-132-fix-v3/c5.18xlarge/nodes/5/dbs/m4.4xlarge/sub-test-comparison/5+x+c5.18xlarge+Jira%2C+m4.4xlarge+DB%2C+run+3/time-series-chart.html): big 10 minute spike at the beginning, then slowly calming down.

Note that timeline response times include both OK and ERROR actions.

### 2 minutes timeout
[Distro](https://s3-eu-west-1.amazonaws.com/quicksilver-jhwr-cache-ireland/QUICK-132-fix-v2/c5.18xlarge/nodes/5/dbs/m4.4xlarge/sub-test-comparison/distribution-comparison.html): 95% hit the 2 minute timeout, the rest is at 1 minute.
[Timeline](https://s3-eu-west-1.amazonaws.com/quicksilver-jhwr-cache-ireland/QUICK-132-fix-v2/c5.18xlarge/nodes/5/dbs/m4.4xlarge/sub-test-comparison/5+x+c5.18xlarge+Jira%2C+m4.4xlarge+DB%2C+run+1/time-series-chart.html): constant 2 minutes duration.

Interesting 🤔